nanome/_internal/logs.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import os
 import sys
+import graypy
 from dateutil import parser
```

Comment:
I think we can remove dateutil from here and the install_requires

Reply:
Oh yeah, removing dependencies is a big +1